app/containers/MessageActions/index.tsx

```diff
@@ -237,6 +239,23 @@ const MessageActions = React.memo(
 				replyInit(message, false);
 			};
 
+			const handleReplyInDM = async (message: TAnyMessageModel) => {
+				if (message.u.username) {
```

Comment:
We should be a little bit more careful about this here.
I'm not sure what could happen if there's no `username` if the message is from some sort of bot or idk.

Reply:
Sure, I added the `optional chain operator`